### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by mtancoigne**

### CODE REVIEW

**Comment by mtancoigne** [comment]
Hop, cancel...

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
-# first-game
-First project of the team
+# First game
+This is the first project of the team: a small game made with [Unity3d](http://unity3d.com). The goal is to test our ability to work together, to learn the Git basics and write code.
+
+The choice of Unity is a good compromise to start coding in C# and to get something done quickly.
+
+## Story
+The game's story can be summed up as follow:
```
Comment (HKellali) [inline README.md]: Follows*

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+  - Tux, the mighty opensource defender is stuck in a computer world ruled by a terrible AI: Cortanouille.
+  - The AI was developped by Megahard, and was its goal is to convert people to the Door, an OS who sucks _your data_...
```
Comment (HKellali) [inline README.md]: ...and its goal is.... an OS that\* sucks....

Comment (mtancoigne) [reply README.md]: Great, i'm happy you're here :)

File: README.md [hunk 3]
```diff
@@ -1,2 +1,12 @@
...
+
+## How can I participate ?
+For now, we will only accept PRs from the team, as the project is meant to train us, but if you're interested, you still can submit issues and comment things. We have to learn, but we like comments...
```
Comment (HKellali) [inline README.md]: ...the project is meant to help us practice... you can still.... comment on things....

**Pull request merged**